make/photon/clair-adapter/Dockerfile.base

```diff
@@ -1,7 +1,5 @@
 FROM photon:2.0
 
-RUN tdnf install -y sudo >>/dev/null\
-    && tdnf clean all \
```

Comment:
why remove the clean all and mkdir?

Reply:
If you look down another line in the diff, it doesn't remove the `mkdir` - it just becomes the first command in the `RUN` statement, which is why it's a changed line.

The `tdnf clean all` gets removed because it's purpose is to clean up extra cache, etc files that `tdnf install` leaves behind.  If we don't invoke `tdnf install`, we don't need to invoke `tdnf clean all`.